mvpa2/tests/test_datasetfx.py

```diff
@@ -139,6 +142,80 @@ def test_sequence_stat(self):
             # TODO: check the content
 
 
+    def test_prt():
+        def give_data():
+            y_axis = np.hstack([[i]*10 for i in range(15)])/100.
+            x_axis =  np.array(range(10)*15)
+            vals = (x_axis+y_axis).reshape(15,10)
+            ds = dataset_wizard(vals, targets=range(15))
+            ds.fa['features'] = range(10)
+            return ds
+
+        def starts_with_digit(x):
+            if len(x) > 0:
+                if x[0] in "0123456789":
+                    return True
+            return False
+
+        def correct_alignment(table, func):
+            for row in table:
+                    if starts_with_digit(row):
+                        row = row[np.array([True if starts_with_digit(e)
+                                  else False for e in row])]
+                        if row[0] == '10':
+                            assert_true(np.all([func(e, '1') or func(e, '10')
+                                                for e in row]))
+                        else:
+                            assert_true(np.all([func(e, row[0]) for e in row]))
+
+        ds = give_data()
+        old_ds = ds.copy()
+        table = ds.to_table()
+
+        correct_alignment(table, str.endswith)
+        correct_alignment(table.transpose(), str.startswith)
+
+        # Test sa selection
+        ds.sa['sa2'] = range(10, 25)
+        assert_false(np.all(ds.to_table(sa='targets')[:,0] != ds.to_table(
+                            sa='sa2')[:,0]))
+        assert_array_equal(ds.to_table(sa='targets')[:,1:],
+                           ds.to_table(sa='sa2')[:,1:])
+
+        # Test alignment again, with more sa keys
+        ds.sa['sa2'] = range(15)
+        table = ds.to_table()
+        correct_alignment(table, str.endswith)
+        correct_alignment(table.transpose(), str.startswith)
+
+        # Test fa selection
+        ds.fa['fa2'] = range(10, 20)
+        assert_false(np.all(ds.to_table(fa='features')[0,:] != ds.to_table(
+                            fa='fa2')[0,:]))
+        assert_array_equal(ds.to_table(fa='features')[1:,:],
+                           ds.to_table(fa='fa2')[1:,:])
+
+        # Test alignment with more fa keys
+        ds.fa['fa2'] = range(10)
+        table = ds.to_table(fa='fa2')
+        correct_alignment(table, str.endswith)
+        correct_alignment(table.transpose(), str.startswith)
+
+        # Test multiple selection
+        ds.sa['sa3'] = range(15)
+        ds.fa['fa3'] = range(10)
+        assert_not_equal(ds.to_table(sa=['sa2', 'sa3'], fa=['fa2','fa3']).shape,
+                         ds.to_table().shape)
+
+        # more fa + sa keys
+        table = ds.to_table()
+        correct_alignment(table, str.endswith)
+        correct_alignment(table.transpose(), str.startswith)
+
+        # Check if ds stays intact
+        table = ds.to_table(n_digits=1)
+        assert_array_equal(ds.samples, old_ds.samples)
+
```

Comment:
would be nice also to add a smoke test which would sweep through all test datasets and see if code doesn't blow. smth like
```python

@sweepargs(ds=datasets.itervalues())
def test_to_table_test_datasets(ds):
    dstxt = ds.to_table()
    for a in itertools.chain(ds.sa.keys(), ds.fa.keys(), ds.a.keys()):
         assert(a in dstxt)
```